05_stacks_queues/lib/stack.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const LinkedList = require('../../04_linked_list/lib/linked_list');
+
+/********************************************************************************
+*         stack data structure module                                           *
+********************************************************************************/
+
+class Stack {
+
+  constructor(){
+    this.storage = new LinkedList();

```

Comment:
Stack should have its own size property.

Reply:
The reason Stack and Queue need their own `size` property is because a user of this class will only have access to the `Stack` interface, and not the `Linked List` underneath. We want the user to be able to access a stack's size by saying `stack.size` instead of `stack.storage.size`, since the user isn't supposed to know about the underlying storage (the `Linked List`).